### PR TITLE
Nclsup 834

### PR DESCRIPTION
**Pull request by janinko**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/pnc/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

**Comment by janinko** [comment]
retest this please

File: facade/src/main/java/org/jboss/pnc/facade/deliverables/DeliverableAnalyzerManagerImpl.java [hunk 1]
```diff
@@ -241,39 +269,103 @@ private org.jboss.pnc.model.Artifact getPncArtifact(Artifact art) {
...
         }
 
-        return builder.build();
+        return builder;
     }
 
-    private String fill(MavenArtifact mavenArtifact) {
+    private static String createDeployPath(MavenArtifact mavenArt) {
+        String filename = createFileName(mavenArt);
+        String deployPath = mavenArt.getGroupId().replace('.', '/') + "/" + mavenArt.getArtifactId() + "/"
```
Comment (pkocandr) [inline facade/src/main/java/org/jboss/pnc/facade/deliverables/DeliverableAnalyzerManagerImpl.java:320]: deployPath should start with a slash

File: facade/src/main/java/org/jboss/pnc/facade/deliverables/DeliverableAnalyzerManagerImpl.java [hunk 2]
```diff
@@ -241,39 +269,103 @@ private org.jboss.pnc.model.Artifact getPncArtifact(Artifact art) {
...
+        filename += "." + mavenArt.getType();
+        return filename;
+    }
+
+    private String createBrewOriginURL(MavenArtifact mavenArt, String nvr) {
+        String brewContentUrl = globalConfig.getBrewContentUrl();
+
+        Matcher matcher = NVR_PATTERN.matcher(nvr);
+        if (!matcher.matches()) {
+            throw new IllegalArgumentException("NVR " + nvr + " does not match expected format.");
```
Comment (pkocandr) [inline facade/src/main/java/org/jboss/pnc/facade/deliverables/DeliverableAnalyzerManagerImpl.java:339]: Wouldn't it be better to log an error and return null instead? Missing the originUrl is not a big deal - we don't really use it - and can be corrected later.

**Review by pkocandr — CHANGES_REQUESTED** [review]
The missing slash at the beginning of the deployPath for Maven artifacts should be corrected. The rest are just details and don't really make much difference IMO, but would be nice to get them too.

File: facade/src/main/java/org/jboss/pnc/facade/deliverables/DeliverableAnalyzerManagerImpl.java (hunk 2 above)
Comment (janinko) [reply facade/src/main/java/org/jboss/pnc/facade/deliverables/DeliverableAnalyzerManagerImpl.java:339]: this shuldn't really happen at all, unless there is a big bug somewhere

**Pull request merged**